src/transformers/generation_logits_process.py

```diff
@@ -520,3 +520,49 @@ def __call__(
             scores[batch_idx * group_size : (batch_idx + 1) * group_size] -= self._diversity_penalty * token_frequency
 
         return scores
+
+
+class ForcedBosTokenLogitsProcessor(LogitsProcessor):
```

Comment:
Nice!

Reply:
(nit) could this be `ForcedBOSToken...` instead of `Bos`?